www/src/components/guidelines/avatar.js

```diff
@@ -1,16 +1,18 @@
-import styled from "@emotion/styled"
+/** @jsx jsx */
+import { jsx } from "theme-ui"
 
-import { Box } from "./system"
-
-const Avatar = styled(Box)()
-
-Avatar.defaultProps = {
-  bg: `grey.10`,
-  borderRadius: 6,
-  flex: `0 0 auto`,
-  height: `avatar`,
-  lineHeight: `solid`,
-  width: `avatar`,
-}
+const Avatar = ({ ...rest }) => (
```

Comment:
Can we turn this to a function component?

Reply:
Working on it